tidb-cloud/secure-connections-to-serverless-tier-clusters.md

```diff
@@ -130,38 +108,58 @@ host: '<host>', port: 4000,user: '<username>', password: '<your_password>', data
 </div>
 </SimpleTab>
 
-## Where is the CA root path on my system?
+## Create and Reset Serverless database password
+
+When you connect to your cluster for the first time, click "Create Password" to automatically generate a password with a length of 16 characters, including numbers, uppercase and lowercase characters, and special characters. The password will be automatically embedded in the sample code for connecting to your cluster easily .
+
+You should copy the password after you create it, and only reset the password after leaving the connect page. Since the serverless cluster is accessed through the Internet,if you need to use the password elsewhere, every time you click on your cluster connection, you need to reset the password of your database cluster to ensure the security and compliance of the database password.
+
+## Root Digital certificate management of Serverless Tier 
+
+### Root Digital certificate‘s issuance and validity  
 
-The following lists the CA root paths on common platforms.
+TiDB Serverless Tier uses certificates from [Let's Encrypt](https://letsencrypt.org/) as a Certificate Authority (CA) for TLS connection between clients and TiDB Serverless Tier clusters. Usually, the root certificate ([ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt)) of Let's Encrypt is present in your system's root CA stores.Let's Encrypt issues Serverless Tier Server digital certificates that are valid for 90 days and will be automatically rotated upon expiration.
+
+TiDB Serverless Tier does not provide a CA root certificate download, because we don't guarantee that the same CA will be used to issue a certificate in the future, which will cause the CA root certificate to change.
+
+If the client uses the system's root CA stores by default, such as Java and Go, you can easily connect securely to TiDB Serverless Tier clusters without specifying the path of CA roots.
+
+However, some drivers and ORMs do not use the system root CA stores. In those cases, you need to configure the CA root path of the drivers or ORMs to your system root CA stores. For example, when you use [mysqlclient](https://github.com/PyMySQL/mysqlclient) to connect a TiDB Serverless Tier cluster in Python on macOS, you need to set `ca: /etc/ssl/cert.pem` in the `ssl` argument.
+
+If you are using a GUI client, such as DBeaver, which does not accept a certificate file with multiple certificates inside, you must download the [ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt) certificate.
```

Comment:
Hmm, seems the Mozilla CA bundle information is not mentioned. Why?

Reply:
windows location , have the same description ,so delete the same content. please check

Reply:
This is not limited to Windows. For any platforms which doesn't carry Let's Encrypt CA could use Mozilla CA bundle. And Windows is the only popular platform we know.

Reply:
Updated，pLease Check